building/build-debs/homeworld-admin-tools/inner-build.sh

```diff
@@ -18,6 +18,10 @@ then
     echo >>src/resources/GIT_VERSION
 fi
 
+echo -n "$APT_REPOSITORY" >>src/resources/APT_REPOSITORY
```

Comment:
Is there a reason to have this file lack a trailing newline?

Reply:
There isn't, but only because rstrip is done on the Python side. I'll remove it the flag here.